src/debug/di/module.cpp

```diff
@@ -3391,6 +3391,49 @@ mdSignature CordbILCode::GetLocalVarSigToken()
     return m_localVarSigToken;
 }
 
+HRESULT CordbILCode::CreateNativeBreakpoint(ICorDebugFunctionBreakpoint **ppBreakpoint)
+{
+    PUBLIC_REENTRANT_API_ENTRY(this);
```

Comment:
This should not be a public API

Reply:
Thanks, it was there because it was public on the first version but now isn't.